JavaScript/2-between.js

```diff
@@ -3,13 +3,15 @@
 // Implementation
 
 const between = (str, quotes) => {
```

Comment:
Looks like we can make in more flexible
```suggestion
const between = (str, start, end) => {
```